commands/slashcommands.go

```diff
@@ -81,10 +81,10 @@ func (p *Plugin) updateGlobalCommands() {
 
 	logger.Info("Slash commands changed, updating....")
 
-	ret, err := common.BotSession.BulkOverwriteGlobalApplicationCommands(common.BotApplication.ID, result)
-	// ret, err := common.BotSession.BulkOverwriteGuildApplicationCommands(common.BotApplication.ID, 614909558585819162, result)
+	// ret, err := common.BotSession.BulkOverwriteGlobalApplicationCommands(common.BotApplication.ID, result)
+	ret, err := common.BotSession.BulkOverwriteGuildApplicationCommands(common.BotApplication.ID, 778487633205788712, result)
```

Comment:
why is this value 778487633205788712 for guildID parameter there?

Reply:
That was my test guild, whoops 😅 will remove